examples/predict.py

```diff
@@ -18,13 +18,15 @@
 model_save_dir = "{}/model/".format(os.getcwd())
 
 data_save_path_test = "{}/image_list_test.csv".format(os.getcwd())
+path_save_dir = "{}/predictions/".format(os.getcwd())
```

Comment:
this line creates a string variable containing current path, appended by `/predictions/`